cc3d/player5/ViewManager/SimpleViewManager.py

```diff
@@ -684,8 +684,8 @@ def __check_update(self):
         self.check_version(check_interval=-1, display_no_update_info=True)
 
     def __open_demo_menu(self):
-        self.demo_browser = ref(DemoBrowser(parent=self))
-        self.demo_browser().exec_()
+        self.demo_browser = DemoBrowser(parent=self)
+        self.demo_browser.exec_()
```

Comment:
Why is this preferred? I used a soft link so that the DemoBrowser could be garbage-collected after it is closed. There is no guarantee that QT handles all garbage collection well after exiting that panel.

Reply:
This is really minor issue. The syntax is simpler without using weakref and as far as garbage collection this is not a big deal either. I am fine with either way though

Reply:
Thanks for putting it back. The extra bit of RAM we save matters to me. 😀